core/operations/document_get_any_replica.hxx

```diff
@@ -55,7 +55,8 @@ struct get_any_replica_request {
               if (ec) {
                   return h(response_type{ make_key_value_error_context(ec, id) });
               }
-              using handler_type = std::function<void(response_type)>;
+              using handler_type = utils::movable_function<void(response_type)>;
```

Comment:
Is there a missing include (i.e. #include "core/utils/movable_function.hxx")?

Reply:
it is probably, because of test include everything it didn't fail here. I will fix it.